src/ts/components/account/LoginFooter.tsx

```diff
@@ -0,0 +1,42 @@
+import React from 'react'
+import t from '../../services/i18n'
+import { Footer } from '../basic-components'
+import { colorSchemes } from '../styles/themes'
+import { IAppState } from '../../background/store/all'
+import { connect } from 'react-redux'
+
+const LoginFooter = (props) => {
+  const styles = {
+    loginFooter: {
+      width: '40%',
+      height: 25,
+      fontSize: 11,
+      margin: 'auto',
+      display: 'flex',
+      justifyContent: 'space-around'
+    },
```

Comment:
this loginFooter can move to `export const Footer = styled.div`? then we just need 1 div